clients/StateChannelWallet.ts

```diff
@@ -52,10 +52,13 @@ export class StateChannelWallet {
     this.hashStore = new Map<string, Uint8Array>();
     this.entrypointAddress = params.entrypointAddress;
     this.scwAddress = params.scwAddress;
+    this.ownerAddress = new ethers.Wallet(params.signingKey).address;
     this.chainProvider = new ethers.JsonRpcProvider(params.chainRpcUrl);
-    this.peerBroadcastChannel = new BroadcastChannel(this.scwAddress + "-peer");
+    this.peerBroadcastChannel = new BroadcastChannel(
+      this.ownerAddress + "-peer",
+    );
     this.globalBroadcastChannel = new BroadcastChannel(
-      this.scwAddress + "-global",
+      this.ownerAddress + "-global",
```

Comment:
I switched this to use the off chain address (corresponding to signing key), rather than the address of the on chain smart contract wallet. Not sure what the pros and cons are, but this lets me make progress without any on chain stuff needing to be in place.